src/components/Step/Step.tsx

```diff
@@ -0,0 +1,204 @@
+import { styled } from '@mui/material/styles'
+import Stack from '@mui/material/Stack'
+import Stepper from '@mui/material/Stepper'
+import Step from '@mui/material/Step'
+import StepLabel from '@mui/material/StepLabel'
+import StepConnector, {
+  stepConnectorClasses
+} from '@mui/material/StepConnector'
+import { StepIconProps } from '@mui/material/StepIcon'
+import CircleIcon from '@mui/icons-material/Circle'
+import { Box } from '@mui/material'
+import DoneAllIcon from '@mui/icons-material/DoneAll'
+import OrderNew from '../../icons/orderNew.svg?react'
+import OrderInProgres from '../../icons/orderInProgres.svg?react'
+import OrderShipped from '../../icons/orderShipped.svg?react'
+import CloseIcon from '@mui/icons-material/Close'
+import { ElementType, FC } from 'react'
+import { Typography } from '../UI/Typography'
+
+export type Status =
```

Comment:
i guess this already existed from BE types

Reply:
and i might be `enum`

Reply:
done